verifier/src/certs.rs

```diff
@@ -56,20 +59,50 @@ pub struct VerifiedCertificate<'a> {
 }
 
 impl<'a> UnverifiedCertificate<'a> {
-    /// Verify the certificate signature.
-    pub fn verify(self, key: &VerifyingKey) -> Result<VerifiedCertificate<'a>> {
-        let tbs_length = self.certificate.tbs_certificate.encoded_len()?;
-        let tbs_size = u32::from(tbs_length) as usize;
-        let tbs_contents = &self.der_bytes[TBS_OFFSET..tbs_size + TBS_OFFSET];
-        key.verify(tbs_contents, &self.signature)
-            .map_err(|_| Error::SignatureVerification)?;
+    /// Verify the certificate signature and time are valid.
+    ///
+    /// # Arguments
+    /// - `key` - The public key to verify the certificate signature with
+    /// - `unix_time` - The duration since UNIX_EPOCH. This value can have
+    ///   `nanos`.
+    pub fn verify(
+        self,
+        key: &VerifyingKey,
+        unix_time: Duration,
```

Comment:
This looks like it should take an `Instant` instead of a `Duration` but I'm pretty sure that's just Rust time API weirdness...

Reply:
In order to support `Instant` we would need to bring some kind of time capability into the enclave, however there isn't a trusted way to do this.  Instead we have the time passed so we don't bring in another untrusted interface into the enclave.

Reply:
Updated the docs on unix_time to try and help clarify the usage